kubeflow/chainer-job/README.md

```diff
@@ -0,0 +1,104 @@
+# chainer-job
+
+Prototypes for running Chainer jobs.
+
+* [Quickstart](#quickstart)
+* [Using Prototypes](#using-prototypes)
+  * [io.ksonnet.pkg.chainer-operator](#io.ksonnet.pkg.chainer-operator)
+  * [io.ksonnet.pkg.chainer-job-simple](#io.ksonnet.pkg.chainer-job-simple)
+  * [io.ksonnet.pkg.chainer-job](#io.ksonnet.pkg.chainer-job)
+
+## Quickstart
+
+*The following commands use the `io.ksonnet.pkg.chainer-job` prototype to generate Kubernetes YAML for chainer-job, and then deploys it to your Kubernetes cluster.*
+
+First, create a cluster and install the ksonnet CLI (see root-level [README.md](../../README.md)).
+
+If you haven't yet created a [ksonnet application](https://ksonnet.io/docs/tutorial#1-initialize-your-app), do so using `ks init <app-name>`.
+
+Finally, in the ksonnet application directory, run the following:
+
+```shell
+# Expand prototype as a Jsonnet file, place in a file in the
+# `components/` directory. (YAML and JSON are also available.)
+$ ks pkg install kubeflow/chainer-job@master
+$ ks prototype use io.ksonnet.pkg.chainer-operator chainer-operator \
```

Comment:
The kubeflow docs mostly use `ks generate` which is just an alias of `ks prototype use`.

Reply:
Thanks for pointing out!  I will change this.